### ISSUE TITLE
NPM install Cheerio dependency version issue

### ISSUE THREAD

**Issue by rutberg** [ksvan/node-red-contrib-komfovent#12]
Hello, 

When trying to install this in a docker container with Node Red I get the following output from NPM;

```
npm info it worked if it ends with ok
npm info using npm@5.0.3
npm info using node@v8.1.3
npm http fetch GET 304 https://registry.npmjs.org/node-red-contrib-komfovent 1333ms (from cache)
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
[..................] | fetchMetadata: sill resolveWithNewModule node-red-contrib-komfovent@0.3.1 checki
npm http fetch GET 304 https://registry.npmjs.org/package.json 305ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/request 315ms (from cache)
[..................] - fetchMetadata: http fetch GET 304 https://registry.npmjs.org/request 315ms (fromnpm http fetch GET 304 https://registry.npmjs.org/cheerio 337ms (from cache)
[..................] - fetchMetadata: http fetch GET 304 https://registry.npmjs.org/cheerio 337ms (from[..................] | fetchMetadata: http fetch GET 304 https://registry.npmjs.org/cheerio 337ms (from[..................] | fetchMetadata: http fetch GET 304 https://registry.npmjs.org/cheerio 337ms (fromnpm http fetch GET 304 https://registry.npmjs.org/cheerio 107ms (from cache)
[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching ve[ .................] - fetchMetadata: sill fetchPackageMetaData error for cheerio@^1.0.0 No matching venpm http fetch GET 200 https://registry.npmjs.org/node-red 1684ms
npm ERR! code ETARGET
npm ERR! notarget No matching version found for cheerio@^1.0.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'node-red-contrib-komfovent'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /usr/src/node-red/.npm/_logs/2019-05-24T10_43_04_157Z-debug.log
``` 

I've successfully pulled the latest cheerio manually, `1.0.0-rc.3` but the install seems to be trying to find `1.0.0` instead of `1.0.0-rc.3`

I'm truly excited to get this running. Any suggestion on where I should look to get it working?

**Comment by ksvan** [comment]
Yes, I can see it. Working dependency should be in this repo master branch now. Trying to get a fix out and publish at npmjs this weekend

**Comment by ksvan** [comment]
Published minor update 0.3.2 to npmjs right now. Issue seems to be solved here. Would be great if you could check and comment back, quite a niche package, so not too many others testing or providing any feedback:)

**Comment by ksvan** [comment]
Issue seems resolved across systems here, no feedback to the contrary received.
